metaflow/plugins/secrets/aws_secrets_manager_secrets_provider.py

```diff
@@ -0,0 +1,165 @@
+import base64
```

Comment:
I didn't review this file. I would just move it. I don't think it belongs in the `secrets` directory.